lib/web/apiserver.go

```diff
@@ -1545,22 +1545,30 @@ func (h *Handler) ping(w http.ResponseWriter, r *http.Request, p httprouter.Para
 		return nil, trace.Wrap(err)
 	}
 
+	group := r.URL.Query().Get(webclient.AgentUpdateGroupParameter)
+
 	return webclient.PingResponse{
 		Auth:              authSettings,
 		Proxy:             *proxyConfig,
 		ServerVersion:     teleport.Version,
 		MinClientVersion:  teleport.MinClientVersion,
 		ClusterName:       h.auth.clusterName,
 		AutomaticUpgrades: pr.ServerFeatures.GetAutomaticUpgrades(),
-		AutoUpdate:        h.automaticUpdateSettings184(r.Context()),
+		AutoUpdate:        h.automaticUpdateSettings184(r.Context(), group, "" /* updater UUID */),
```

Comment:
nit: why include the inlay comment? feels more like an IDE setting than something that should be part of the code (in vscode `Inlay Hints: Parameter Names`). (same comment in quite a few palces)

Reply:
Got the opposite feedback in a PR last time (from @codingllama I think?). My IDE displays the parameter name but I can understand people who want the code to be readable out of the box. I don't have a strong opinion on this but we should pick one solution and be consistent.

Reply:
I'm +1 for the inline comments, I'm often reading code outside of IDE (Github, for example).